### ISSUE TITLE
MariaDB refuses access to photoprism

### ISSUE THREAD

**Issue by catleeball** [photoprism/photoprism#1173]
Hi PhotoPrism team,

When upping PhotoPrism for the first time from the [example docker-compose.yaml](https://dl.photoprism.org/docker/docker-compose.yml) with only small modifications to paths and updating passwords, the MariaDB container refuses to accept connections from the photoprism container, getting `Error 1045: Access denied`.

- [commands and logs](https://gist.github.com/catleeball/a08c4a7e85fff73183969497a7c3985b)
- my [docker-compose.yaml](https://gist.github.com/catleeball/b92fb98d90080a1dcd0a64c7ec8a13de)
- output of [`exec photoprism config`](https://gist.github.com/catleeball/f32a6be2cf7cfef886b3fd0390c5d421)
- `uname -a`: `Linux bip 5.8.0-48-generic #54~20.04.1-Ubuntu SMP Sat Mar 20 13:40:25 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux`

Using sqlite seems to work fine. I've tried upping the containers both as my normal user and as root via sudo, but same issue regardless.

Please let me know if you have any ideas or need any more info. Thanks!

**Comment by kyeb** [comment]
I ran into this issue when first trying to install yesterday. For me, the problem was that when I first ran `docker-compose up`, I had modified the passwords in `docker-compose.yaml` to not match. Restarting the container at that point will no longer help, because the DB stores the password on disk.

Destroying the containers (`docker ps -a` shows all, then `docker rm <container name>` and starting from scratch with the passwords in `PHOTOPRISM_DATABASE_PASSWORD` and `MYSQL_PASSWORD` matching worked for me.
